src/__tests__/hover.js

```diff
@@ -38,3 +38,87 @@ test('no events fired on labels that contain disabled controls', () => {
     `No events were fired on: label`,
   )
 })
+
+test('fires non-bubbling events on parents for hover', () => {
+  const wrapper = document.createElement('div')
+  wrapper.innerHTML = '<div><button></button></div>'
+  document.body.append(wrapper)
+
+  const calls = []
+  function addListeners(el) {
+    for (const event of [
+      'mouseenter',
+      'mouseover',
+      'mouseleave',
+      'mouseout',
+      'pointerenter',
+      'pointerover',
+      'pointerleave',
+      'pointerout',
+    ]) {
+      addEventListener(el, event, () => {
+        calls.push(`${el.tagName}: ${event}`)
+      })
+    }
+  }
+  const div = wrapper.firstChild
+  const button = div.firstChild
```

Comment:
Could we add a comment here explaining why `setup` isn't being used like it is for the rest of the tests?

Reply:
Done, I did change it out for `setup` though and added an explanation why I didn't use `getEventSnapshot`.